core/spec/models/spree/store_spec.rb

```diff
@@ -37,19 +37,32 @@
   end
 
   describe ".default" do
-    let!(:store)    { create(:store) }
-    let!(:store_2)  { create(:store, default: true) }
+    before do
+      Spree::Store.delete_all
```

Comment:
okeydoke, I'll delete it. I still have no idea how a bunch of other tests (especially controller/feature tests) are managing to pass _without_ any apparent creation of a spree default store, now that orders require stores. I guess something must be creating a spree default store somewhere else I haven't noticed.